src/main.rs

```diff
@@ -291,12 +288,12 @@ struct CheckRelease {
     verbosity: clap_verbosity_flag::Verbosity<clap_verbosity_flag::InfoLevel>,
 
     /// Whether to print colors to the terminal:
-    /// always, alwaysansi (always use only ANSI color codes),
+    /// always, always-ansi (always use only ANSI color codes),
     /// auto (based on whether output is a tty), and never
     ///
     /// Default is auto (use colors if output is a TTY, otherwise don't use colors)
```

Comment:
I think you can remove the docstring on this field.

Reply:
Interesting, why remove it? Will clap automatically display the `--color` options in `--help` if we do that?

The `--color` options are non-obvious and I think it's important to have a help string in the CLI for them.

Reply:
The `colorchoice_clap` has a docstring for the `color` field that gets picked up by clap when used with `flatten`. It looks something like the following:

```
      --color <WHEN>  Controls when to use color [default: auto] [possible values: auto, always, never]
```

Reply:
Ah neat! This wasn't obvious to me from the `colorchoice_clap` docs, so perhaps best to leave a regular (non-doc) comment next to the field indicating we expect the built-in `--help` text there.